change/@ni-nimble-components-54ec9080-33bb-47a6-a37d-d926c42de859.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you make sure the tech debt item includes a note about adding test coverage for the case that broke?